tensorboard/plugins/graph/tf_graph_controls/tf-graph-controls.html

```diff
@@ -799,45 +809,59 @@
       value: null,
       type: Object,
       notify: true,
+      // TODO(stephanwlee): FIX this to readOnly and fix the setter.
       readonly: true,
     },
     colorBy: {
       type: String,
       value: 'structure',
       notify: true,
-      readonly: true
     },
     colorByParams: {
       type: Object,
       notify: true,
+      // TODO(stephanwlee): FIX this to readOnly and fix the setter.
```

Comment:
What does this comment mean?

Reply:
It is a self reminder to fix this in a separate PR. In case it still is not clear, there is a typo here (readonly -> readOnly) and the property is actually writeable.

Reply:
Got it. I didn’t realize that `readonly` was a typo. Thanks.